Client/Frontend/Browser/ButtonToast.swift

```diff
@@ -133,16 +124,54 @@ class ButtonToast: Toast {
             make.top.equalTo(toastView.snp.top)
             make.height.equalTo(ButtonToastUX.ToastHeight)
         }
-
         return toastView
     }
-
-    @objc func buttonPressed(_ gestureRecognizer: UIGestureRecognizer) {
-        completionHandler?(true)
-        dismiss(true)
+    
+    func setupPaddedButton(stackView: UIStackView, buttonText:String?) {
```

Comment:
Not sure why `buttonText` is optional, wouldn't the button be empty without text?

Reply:
Refactored the code, so now just like before we don't add the button if there is no button text.